.github/workflows/build_docs.yml

```diff
@@ -29,6 +29,8 @@ jobs:
           pip install nbsphinx
           pip install sphinx_rtd_theme
           pip install jupyter
+          pip install numpydoc
```

Comment:
I guess numpydoc isn't required in this CI file anymore?

Reply:
removed!